package.json

```diff
@@ -26,9 +26,9 @@
   },
   "devDependencies": {
     "@codechecks/build-size-watcher": "^0.1.0",
-    "@codechecks/client": "^0.1.4",
-    "conditional-type-checks": "^1.0.1",
-    "prettier": "^1.18.2",
-    "typescript": "^3.5.2"
+    "@codechecks/client": "^0.1.10",
+    "conditional-type-checks": "^1.0.4",
+    "prettier": "^1.19.1",
+    "typescript": "^3.7.2"
```

Comment:
one more thing: you need to also update TS version in the `peerDependencies` above

Reply:
Done.